docs/content/dagster-cloud/insights/asset-metadata.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Using asset metadata with Dagster Cloud Insights | Dagster Docs"
+description: "Integrating external metrics with Dagster Insights."
+
+platform_type: "cloud"
+---
+
+# Using asset metadata with Dagster Cloud Insights (Experimental)
+
+<Note>
+  This feature is considered <strong>experimental</strong>.
+</Note>
+
+Any numeric [asset materialization metadata](/concepts/assets/software-defined-assets#recording-materialization-metadata) attached to your assets can be viewed and aggregated in the Dagster Insights UI. This is a good way to track usage, cost, data size, or other characteristics of your data assets over time, and across different parts of your platform.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- A Dagster Cloud account on the Enterprise plan
+- Access to the [Dagster Cloud Insights feature](/dagster-cloud/insights)
```

Comment:
It looks like you removed the **Getting Access** section from the main Insights page - is this feature now available by default? If so, I think we can take this out

Reply:
Yes, it is!